src.repos

```diff
@@ -1,5 +1,5 @@
 repositories:
-  ros_control:
```

Comment:
Why did we put this file here?

Reply:
This is an example to be used by the test script.